### PR TITLE
Add an API to convert to QColor

### PR DESCRIPTION
**Pull request by vallsv**

### CODE REVIEW

File: silx/gui/colors.py [hunk 1]
```diff
@@ -180,6 +180,20 @@ def greyed(color, colorDict=None):
...
 
 
+def asQColor(color):
+    """Convert color code '#RRGGBB' and '#RRGGBBAA' to a `qt.QColor`.
+
+    It also convert RGB(A) values from uint8 to float in [0, 1] and
+    accept a QColor as color argument.
+
+    :param str color: The color to convert
+    :param dict colorDict: A dictionary of color name conversion to color code
```
Comment (payno) [inline silx/gui/colors.py]: documentation not up to date
```suggestion
```

File: silx/gui/colors.py [hunk 2]
```diff
@@ -180,6 +180,20 @@ def greyed(color, colorDict=None):
     return g, g, g, a
 
 
+def asQColor(color):
+    """Convert color code '#RRGGBB' and '#RRGGBBAA' to a `qt.QColor`.
+
+    It also convert RGB(A) values from uint8 to float in [0, 1] and
```
Comment (payno) [inline silx/gui/colors.py]: documentation not up to date
```suggestion
```

File: silx/gui/colors.py [hunk 3]
```diff
@@ -180,6 +180,20 @@ def greyed(color, colorDict=None):
...
+    accept a QColor as color argument.
```
Comment (payno) [inline silx/gui/colors.py]: documentation not up to date
```suggestion
```

**Comment by vallsv** [comment]
I merged one of your changes and discard others, cause uint, float, and QColor are all supported.

Then i updated the documentation for other functions.

**Pull request merged**